src/css/spectrum/spectrum.css

```diff
@@ -13,7 +13,7 @@ License: MIT
     *display: inline;
     *zoom: 1;
     /* https://github.com/bgrins/spectrum/issues/40 */
-    z-index: 9999994;
+    z-index: 20000;
```

Comment:
Ok, so that's the tooltip from the color square "input/button" beneath the spectrum picker. IMO this tooltip should remain below the colorpicker.

However, it's true that the "current" color is not displayed anywhere here. Could be good for a follow up issue.